bin/deploy.sh

```diff
@@ -48,14 +60,19 @@ for region in "${DEIS_REGIONS[@]}"; do
       fi
     fi
 
-    # skip admin apps in eu-west
-    if [[ "$region" == "eu-west" && "$appname" == *admin* ]]; then
+    # skip admin apps in DB read-only region
+    if [[ "$region" != "$DB_RW_REGION" && "$appname" == *admin* ]]; then
       continue
     fi
     NR_APP="${appname}-${region}"
     echo "Pulling $DOCKER_IMAGE_TAG into Deis app $appname in $region"
     ./deis pull "$DOCKER_IMAGE_TAG" -a "$appname"
 
+    if [[ "$region" == "$DB_RW_REGION" && elementIn "$appname" "${DEIS_POST_DEPLOY_APPS[@]}" ]]; then
```

Comment:
There is a syntax error here that I missed the first time through. Testing this locally I'm getting a `conditional binary operator expected error` due to the use of the bash function inside the `[[ ]]`. The following works for me locally:
```sh
if $([ "$region" == "$DB_RW_REGION" ] && elementIn "$appname" "${DEIS_POST_DEPLOY_APPS[@]}"); then
```